sound/soc/sof/sof-audio.h

```diff
@@ -297,6 +307,7 @@ struct snd_sof_pcm_stream {
 	 * active or not while suspending the stream
 	 */
 	bool suspend_ignored;
+	struct snd_sof_pipeline_trigger_info pipeline_info;
```

Comment:
@ranj063 it looks like @RanderWang is about to add a 'private' field here for IPC-specific stuff in https://github.com/thesofproject/linux/pull/3787/commits/490f6530b6df3233464e7784a5c231fddfc09c54#diff-3aa89cf77fc07290f36500245dad57c983ae8628a8fb4b43804391a01ca4f520R295

How many private things are we going to add here?

Reply:
@plbossart there's nothing IPC-specific in this struct and I have to insist that this information belongs in snd_sof_pcm_stream. There's just no place else that this information fits in IMHO

Reply:
The other reason to no move it to a private field @plbossart is the fact that this field is populated in IPC-agnostic sof-audio.c. If I move it inside a private field, I wont be able to do that any more.